app/services/csv/missing_addresses_csv_service.rb

```diff
@@ -0,0 +1,165 @@
+module Csv
+  class MissingAddressesCsvService
+    def initialize(organisation)
+      @organisation = organisation
+    end
+
+    def create_missing_lettings_addresses_csv
+      logs_with_missing_addresses = @organisation.managed_lettings_logs
```

Comment:
are there any logs with missing address data that we don't have a managing org for? I don't _think_ they could have been created if not but thought I'd confirm

Reply:
I think they'd get filtered out in the rest of the query anyway, because you can't have uprn and address values if managing org is not answered

Reply:
Cool, sounds good :)